UMA-Bots/Wallet-Monitoring/src/utils.ts

```diff
@@ -0,0 +1,40 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+export const FILLED_RELAY_EVENT =
+  "event FilledRelay(uint256 amount, uint256 totalFilledAmount, uint256 fillAmount, uint256 repaymentChainId, uint256 originChainId, uint256 destinationChainId, uint64 relayerFeePct, uint64 appliedRelayerFeePct, uint64 realizedLpFeePct, uint32 depositId, address destinationToken, address indexed relayer, address indexed depositor, address recipient, bool isSlowRelay)";
+export const GOERLI_POC_SPOKEPOOL_ADDRESS = "0x8Ec948A7823745FE5e2591df63654C9d7e4809c4";
+export const MAINNET_SPOKEPOOL = "0x4D9079Bb4165aeb4084c526a32695dCfd2F77381";
+export const OPTIMISM_SPOKEPOOL = "0xa420b2d1c0841415A695b81E5B867BCD07Dff8C9";
+export const POLYGON_SPOKEPOOL = "0x69B5c72837769eF1e7C164Abc6515DcFf217F920";
+export const ARBITRUM_SPOKEPOOL = "0xB88690461dDbaB6f04Dfad7df66B7725942FEb9C";
+
+export const GOERLI_POC_MONITORED_ADDRESSES: string[] = [
+  "0x1Abf3a6C41035C1d2A3c74ec22405B54450f5e13",
+  "0x496B53B037BC66Db6e7D34d101FA045e7Bf7D388",
+]; // PoC addresses
+
+export function getFindingInstance(
+  amountStr: string,
+  originChainIdStr: string,
+  destinationChainIdStr: string,
+  depositorStr: string,
+  recipientStr: string,
+  isSlowRelayStr: string
+) {
+  return Finding.fromObject({
+    name: "Monitored Wallet address Used",
+    description: "A monitored wallet address was used to transfer funds across the bridge",
+    alertId: "UMA-10",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "UMA",
+    metadata: {
+      amount: amountStr,
+      originChainId: originChainIdStr,
+      destinationChainId: destinationChainIdStr,
+      depositor: depositorStr,
+      recipient: recipientStr,
+      isSlowRelay: isSlowRelayStr,
+    },
+  });
```

Comment:
not a requested change but you already assign the variable types so I think it's not necessary to rename the variables with `Str` in the end. plus, if you keep the variable names the same with the metadata keys, you can only pass their name without assigning the values to the param names. 
```suggestion
export function getFindingInstance(
  amount: string,
  originChainId: string,
  destinationChainId: string,
  depositor: string,
  recipient: string,
  isSlowRelay: string
) {
  return Finding.fromObject({
    name: "Monitored Wallet address Used",
    description: "A monitored wallet address was used to transfer funds across the bridge",
    alertId: "UMA-10",
    severity: FindingSeverity.Info,
    type: FindingType.Info,
    protocol: "UMA",
    metadata: {
      amount,
      originChainId,
      destinationChainId,
      depositor,
      recipient,
      isSlowRelay,
    },
  });
```

Reply:
Thanks for this trick @0xtaf , it increases the readability 👍